### DISCUSSION TITLE
Translation message for 'email:dns' rule

### DISCUSSION THREAD

**Discussion by pultho** [laravel/framework#36282] [Q&A]
Laravel allows additional settings for the `email` validation rule (like `email:dns` or `email:rfc,dns`).

https://laravel.com/docs/8.x/validation#rule-email

But how to translate those messages? Neither `["email:dns" => "Some message"]` nor `["email" => ["dns" => "Some message]]` works in the `validation.php` file. 

Any idea?

**Comment by zotosk** [comment]
For any one else having trouble with this one, you can do for 
`email:rfc,dns ` or `phone:AUTO,GR,mobile`, tested it works fine

example for `rules`
```
    public function rules(): array
    {
        return [
            'email' => ['required', 'email:rfc,dns'],
            'phone' => ['required', 'phone:AUTO,GR,mobile'],
        ];
    }
```

`lang/el/validation.php`
```
    'custom' => [
        'email' => [
            'email'  => 'Το :attribute πρέπει να είναι μια έγκυρη διεύθυνση email.',
        ],
        'phone' => [
            'phone' => 'Το :attribute πρέπει να είναι ένας έγκυρος αριθμός κινητού τηλεφώνου.',
        ],
    ],
```
This should be enough
